package.json

```diff
@@ -32,6 +32,7 @@
     "bitcore-mnemonic": "=1.1.1",
     "copy-to-clipboard": "=3.0.6",
     "flexboxgrid": "=6.3.1",
+    "history": "^4.7.2",
```

Comment:
This should use `=` instead of `^`.
We should set up this in `.npmrc` file like `lisk-js` guys did https://github.com/LiskHQ/lisk-js/pull/382

Reply:
TBH, I never liked this approach, choosing `=` we easily end up with outdated dependencies.  if there's a breaking change in one of the dependencies, I'll prefer to get notified of somehow. that's exactly the reason I clean install my node modules every once in a while.

Reply:
@reyraa, @slaweet https://github.com/greenkeeperio/greenkeeper

Reply:
Sure it's a good idea, but please don't tell me to add it in this PR :-D